src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -385,20 +389,21 @@ export class HuiAreaCard
       if (!(domain in entitiesByDomain)) {
         return;
       }
-      DEVICE_CLASSES[domain].forEach((deviceClass) => {
+      this._deviceClasses[domain].forEach((deviceClass) => {
         if (
           entitiesByDomain[domain].some(
             (entity) => entity.attributes.device_class === deviceClass
           )
         ) {
-          sensors.push(html`
-            ${DOMAIN_ICONS[domain][deviceClass]
-              ? html`<ha-svg-icon
-                  .path=${DOMAIN_ICONS[domain][deviceClass]}
-                ></ha-svg-icon>`
-              : ""}
-            ${this._average(domain, deviceClass)}
-          `);
+          const icon =
+            DOMAIN_ICONS[domain][deviceClass] ||
+            FIXED_DEVICE_CLASS_ICONS[deviceClass];
```

Comment:
Can we remove `sensor` from `DOMAIN_ICONS`? As they are present in `FIXED_DEVICE_CLASS_ICONS` already.